data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,426 @@ mission "Remnant: Face to Maw 2B"
+				accept
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+
+
+
+mission "Remnant: Crystal Research 6"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Key Stone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling" "Winds of Light"
+		dialog
+			`The scanner chimes softly once again, informing you that it has completed scanning the Winds of Light and has successfully archived the data from this second scanning.`
+	to offer
+		has "Remnant: Crystal Research 5: done"
+	on offer
+		conversation
+			`When you and the <npc> settle back onto the landing pad, there is already a dockworker waiting with a Key Stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the Keystone to a lift and disconnects it from the ship. As the Keystone is hauled up to the roof another lift lowers the replacement Key Stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`
+			`	"Ready? Let's go!" Aeolus signs as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear and watches as the two ships head back out of the dome.`
+				launch
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+		conversation
+			`While in transit, you took the opportunity to examine the preliminary results of the scans: a few differences between the two crystals, but both appear to have functioned similarly. Upon landing on Viminal, Aeolus comes over to collect a copy of the data for the researchers.`
+			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully. "Perhaps more importantly, it may help us understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aid."`
+			`	He looks down at the data crystal. "I have one more task to ask of you. Meet me in the cafeteria in an hour to discuss it. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`
```

Comment:
The term is used earlier in `"Remnant: Expanded Horizons Quarg 2"`. 

While I agree that it could be better to have it introduced, I also suspect that <first> <last> is sufficiently bright to realize a data holding device whatever the form. Case in point, data storage today. 5.25 floppy, 3.5 floppy, CD, DVD, zip drive, omega drive, external hard drive, and USB drives in hundreds of various form factors, including partial connections that don't even make a full USB. I doubt the future has any fewer data storage devices, so I don't see any problem with <first> recognizing a data storage device even if it looks kind of odd.